gc/base/GCExtensionsBase.hpp

```diff
@@ -1105,31 +1105,94 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	MMINLINE MM_Heap* getHeap() { return heap; }
 
-#if defined(OMR_GC_MODRON_SCAVENGER)
 	MMINLINE void
 	setGuaranteedNurseryRange(void* start, void* end)
 	{
+#if defined(OMR_GC_MODRON_SCAVENGER)
 		_guaranteedNurseryStart = start;
 		_guaranteedNurseryEnd = end;
+#endif /* OMR_GC_MODRON_SCAVENGER */
 	}
 
 	MMINLINE void
 	getGuaranteedNurseryRange(void** start, void** end)
 	{
+#if defined(OMR_GC_MODRON_SCAVENGER)
 		*start = _guaranteedNurseryStart;
 		*end = _guaranteedNurseryEnd;
+#elif /* OMR_GC_MODRON_SCAVENGER */
```

Comment:
All`#elif`s will need to be replaced with `#else` or `#elif defined(MACRO_NAME)`. Otherwise, the code will not compile.
```suggestion
#else /* OMR_GC_MODRON_SCAVENGER */
```